src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLConstants.java

```diff
@@ -29,6 +29,23 @@ public final class MapMLConstants {
     /** format MediaType */
     public static final MediaType MAPML_MEDIA_TYPE = new MediaType("text", "mapml", StandardCharsets.UTF_8);
 
+    /** Multilayer represented as multi-extent */
+    public static final String MAPML_MULTILAYER_AS_MULTIEXTENT = "mapmlusemultiextents";
+
+    public static final String MAPML_USE_FEATURES_REP = "mapmlusefeatures";
```

Comment:
What about the     

```
public static final String MAPML_FEATURE_FO = "mapmlfeatures";
```

constant below? Is it still needed?
Also, what does REP stand for? (FO is format option, which made sense, but .. REP?)

Reply:
Changed it to MAPML_CREATE_FEATURE_LINKS for clarity sake.  This argument leads document representation to create feature links using MAPML_FEATURE_FO, which in turn results in the actual XML features.  "REP" stood for representation, but I agree that is unclear.